src/main/java/org/breedinginsight/brapps/importer/services/processors/experiment/ExperimentUtilities.java

```diff
@@ -70,8 +70,7 @@ public class ExperimentUtilities {
     public static final String PREEXISTING_EXPERIMENT_TITLE = "Experiment Title already exists";
     public static final String MISSING_OBS_UNIT_ID_ERROR = "Experimental entities are missing ObsUnitIDs";
     public static final String UNMATCHED_COLUMN = "Ontology term(s) not found: ";
-
-
+    public static final String INVALID_OBS_UNIT_ID_ERROR = "Invalid ObsUnitID";
```

Comment:
The message for empty ids is the same but duplicate ids have a more specific error message. Also, constants specific to the append workflow are [here](https://github.com/Breeding-Insight/bi-api/blob/develop/src/main/java/org/breedinginsight/brapps/importer/services/processors/experiment/model/ExpImportProcessConstants.java).
```suggestion
        public enum ErrMessage {
        MULTIPLE_EXP_TITLES("File contains more than one Experiment Title"),
        MISSING_OBS_UNIT_ID("Invalid ObsUnitID"),
        PREEXISTING_EXPERIMENT_TITLE("Experiment Title already exists"),
        UNMATCHED_COLUMN("Ontology term(s) not found: "),
        OBS_UNIT_NOT_FOUND("Invalid ObsUnitID"),
        DUPLICATE_OBS_UNIT_ID("ObsUnitId is repeated");
```